src/components/Spinner/index.js

```diff
@@ -1,9 +1,12 @@
 import { h } from 'preact'
 import style from './style.css'
+import { compose } from '../utils/func'
+import {localised} from '../../locales'
 
-const Spinner = () => {
+const Spinner = ({translate}) => {
   return (
     <div className={style.loader}>
+      <span className={style.accessibleSpinnerCopy} role="heading" aria-level="1">{translate('loading')}</span>
```

Comment:
Have you tried `aria-label`? Screen readers do read it, this is what I've used for e.g. "close" button that visually has only "x", no text.

Reply:
Yes, I implemented this based on those articles I read but after I reviewed your PR I thought it might be better to just use `aria-label`. I will try it and see if it works well for page transition